content/core-features/authentication.textile

```diff
@@ -373,6 +374,125 @@ The following JavaScript example demonstrates how to issue an "Ably Token":#ably
   })
 ```
 
+h2(#token-revocation). Token revocation
+
+Token revocation is a mechanism that enables an app to invalidate authentication tokens. This invalidation can be used to force specified clients to re-obtain a token, and subsequently enables the app to modify the rights granted to clients, or to decline to re-issue a token.
+
+The main use case of token revocation is as a method to combat malicious clients. In cases where the client behaves safely, you don't need token revocation - your code can instruct the connection to Ably to close (or detach from a channel it shouldn't have access to). But if you want to prevent any chance of a user obtaining the token before it is disconnected, and using it in their own custom client to gain access to Ably, then token revocation can be used.
+
+Without token revocation, in order to restrict potential rogue access, you would make tokens with a short TTL (say 10 minutes) and then simply not issue a new token to a suspect client. Unfortunately, reducing the TTL would increase load on your authentication server, so token revocation can provide a more scalable solution to the issue of rogue clients.
+
+Token revocation has to be enabled for you by Ably. There is a performance cost to enabling this feature. "Contact Ably":https://ably.com/contact if you'd like to enable this feature for a specific application.
+
+h3. Revocable tokens
+
+Tokens are revocable if they are created from an API key that has a @revocable@ attribute set to @true@. 
```

Comment:
The attribute is `revocableTokens` rather than `revocable` (keys themselves are always revocable)

(not that 99% of people will care about the actual raw attribute name, they'll just be ticking a box in the UI, to be decided what that box will be labelled)

Reply:
Changed.